conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -368,6 +373,14 @@ private TypeSpec generateEndpointHandler(
         return endpointBuilder.build();
     }
 
+    private static ClassName createEndpointErrorException(EndpointError endpointError, Options options) {
```

Comment:
Can we share this between `CheckedErrorGenerator.errorExceptionsClassName` and this class? Perhaps in ErrorGenerationUtils?